backend/src/v2/objectstore/object_store.go

```diff
@@ -351,14 +366,10 @@ type SessionInfo struct {
 	SecretKeyKey string
 }
 
-func createBucketSession(ctx context.Context, namespace string, sessionInfo *SessionInfo, client kubernetes.Interface) (*session.Session, error) {
+func CreateBucketSession(sessionInfo *SessionInfo, creds *credentials.Credentials) (*session.Session, error) {
```

Comment:
This function doesn't need to be public.

Reply:
addressed